debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectModel.java

```diff
@@ -479,5 +479,18 @@ public static boolean isInlineContiguousArraylet(J9IndexableObjectPointer arrayP
 	{
 		return gcObjectModel.isInlineContiguousArraylet(arrayPtr);
 	}
+
+	/**
+	 * Determine the validity of the data address belonging to arrayPtr.
+	 *
+	 * @param arrayPtr array object who's data address validity we are checking
+	 * @throws CorruptDataException if there's a problem accessing the indexable object dataAddr field
+	 * @throws NoSuchFieldException if the indexable object dataAddr field does not exist on the build that generated the core file
+	 * @return true if the data address of arrayPtr is valid, false otherwise
+	 */
+	public static boolean isCorrectDataAddrPointer(J9IndexableObjectPointer arrayPtr) throws CorruptDataException, NoSuchFieldException
+	{
+		return gcObjectModel.isCorrectDataAddrPointer(arrayPtr);
+	}
```

Comment:
Perhaps "has" is better prefix than "is", i.e. `hasCorrectDataAddrPointer()`; the argument is not the data address pointer, but rather the array (whose data address pointer we seek to validate).

Reply:
>There are other compile errors besides the one I pointed out.
Yes, please see [this](https://github.com/eclipse-openj9/openj9/pull/14667#discussion_r1194359952) comment for more details.